app/models/candidate_interface/create_account_or_sign_in_form.rb

```diff
@@ -2,9 +2,12 @@ module CandidateInterface
   class CreateAccountOrSignInForm
     include ActiveModel::Model
 
-    attr_accessor :existing_account
+    attr_accessor :existing_account, :email
 
     validates :existing_account, presence: true
+    with_options if: -> { existing_account? } do
+      validates :email, presence: true, email_address: true
+    end
```

Comment:
Is this the same as this?

```suggestion
  validates :email, presence: true, email_address: true, if: -> { existing_account? }
```